pkg/manager/api/event.go

```diff
@@ -32,7 +32,7 @@ type EventManager struct {
 
 var _ model.EventManager = &EventManager{}
 
-func (m *EventManager) Emit(ctx context.Context, data map[string]interface{}) (*model.Event, error) {
+func (m *EventManager) Emit(ctx context.Context, data map[string]interface{}, key string) (*model.Event, error) {
```

Comment:
The key needs to be added to the `model.Event` struct returned by this function, too. Looks like it did get updated everywhere else.